### PR TITLE
Docs: Fix HTTP Request example

### PR DESCRIPTION
**Pull request by ruben1**
Changes: 
1.- On the "data" event in the response, wait for all the data to arrive and append that data to an empty string
2.- On the "end" event in the response, console log the body response after it has been appended to an empty string in the "data" event

### CODE REVIEW

File: doc/api/http.markdown [hunk 1]
```diff
@@ -500,9 +500,13 @@ Example:
       console.log('STATUS: ' + res.statusCode);
       console.log('HEADERS: ' + JSON.stringify(res.headers));
       res.setEncoding('utf8');
+      var str = "";
```
Comment (cjihrig) [inline doc/api/http.markdown]: This should use single quotes like the rest of the example.

File: doc/api/http.markdown [hunk 2]
```diff
@@ -500,9 +500,13 @@ Example:
...
       res.on('data', function (chunk) {
-        console.log('BODY: ' + chunk);
+        str += chunk;
       });
+      res.on("end", function() {
```
Comment (cjihrig) [inline doc/api/http.markdown]: Single quotes please.

File: doc/api/http.markdown [hunk 3]
```diff
@@ -500,9 +500,13 @@ Example:
...
+        console.log('BODY: ' + str);
+      })
```
Comment (cjihrig) [inline doc/api/http.markdown]: Missing a semicolon

**Comment by cjihrig** [comment]
Please update the commit message to adhere to the guidelines in [CONTRIBUTING.md](https://github.com/joyent/node/blob/master/CONTRIBUTING.md).

**Comment by ruben1** [comment]
Thanks for the feedback. I amended the last commit

**Comment by jasnell** [comment]
@cjihrig .. it looks like a somewhat similar change was already made in nodejs/node that would make this a bit superfluous. I'm inclined to close without further action but wanted to check with you first.

**Comment by cjihrig** [comment]
If the same change already exists on nodejs/node, then I agree that we don't need this. If the change hasn't been made over there, feel free to open this there :-)

**Pull request closed without merging**